Misc/NEWS.d/next/Library/2017-12-21-15-14-41.bpo-28441.8XZw-t.rst

```diff
@@ -0,0 +1,3 @@
+On platforms that have an executable suffix (such as ``.exe`` on
+Windows/Cygwin), ensure that ``sys.executable`` always includes the
```

Comment:
On Windows, `PC/getpathp.c` is used instead of `Module/getpath.c`.
So Windows is not affected by this change.
Cygwin and all other platforms are affected when `configure --with-suffix=.<SUFFIX>` option is used.

Reply:
Agreed.  Would you accept a different wording?